### PR TITLE
chore(website): Taquito domain change

### PR DESCRIPTION
**Pull request by GImbrailo**
Change domain from tezostaquito.io to taquito.io and remove Cloney reference

### CODE REVIEW

**Review by dsawali — APPROVED** [review]

**Review by dsawali — CHANGES_REQUESTED** [review]
looks good but there's some unit tests failing due to the url change

File: docs/dapp_prelaunch.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ author: Claude Barde
 
 #### You will find below a few considerations that you may take into account when building your Tezos dapp or before publishing it. These pointers help you prevent eventual bugs and improve your users' experience.
 
-- Is your dapp using a reliable and scalable RPC node? Running your own RPC node is often the best choice, especially if you anticipate significant traffic to your dapp. Using a paid hosted node is also a great idea, such as Nautilus from Cryptonomic, or TezTools. You can find [here](http://tezostaquito.io/docs/rpc_nodes#list-of-community-run-nodes) a list of public Tezos nodes.
+- Is your dapp using a reliable and scalable RPC node? Running your own RPC node is often the best choice, especially if you anticipate significant traffic to your dapp. Using a paid hosted node is also a great idea, such as Nautilus from Cryptonomic, or TezTools. You can find [here](http://taquito.io/docs/rpc_nodes#list-of-community-run-nodes) a list of public Tezos nodes.
```
Comment (ac10n) [inline docs/dapp_prelaunch.md:8]: This might be a good chance to change the `http` links to `https`

Comment (GImbrailo) [reply docs/dapp_prelaunch.md]: Good catch. Updated!

**Comment by GImbrailo** [comment]
> looks good but there's some unit tests failing due to the url change

@dsawali these tests will fail until the changes have been merged in `master` I have reverted them in this PR and will make a new PR once this one has been merged?

**Review by ac10n — APPROVED** [review]

**Review by dsawali — APPROVED** [review]

**Pull request merged**